travel_platform/travel_platform/settings.py

```diff
@@ -131,3 +131,30 @@
 
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+
+LOGGING = {
```

Comment:
as far as i know from googling, it's quite common to have logging configs in Django config